src/gp_doctest.cpp

```diff
@@ -535,18 +529,18 @@ TEST_CASE("GPInstance: inverted GPCSP probabilities") {
       1.,                            // 0 (rootsplit)
       1.,                            // 1 (rootsplit)
       1.,                            // 2 (rootsplit)
-      1. / 3.,                       // 3
-      0.5,                           // 4
-      1.,                            // 5
+      1.,                            // 3
+      1.,                            // 4
+      2. / 3.,                       // 5
       // We have the 0.5 coming from node 12, but that's split evenly between the two
       // descendants, so we have 0.25 from each. Thus even weights.
       0.5,      // 6
-      1.,       // 7
-      1.,       // 8
+      0.5,      // 7
+      0.5,      // 8
```

Comment:
I don't follow this one: the two nodes are 0.25 and 0.5. 

I'm sure you are right though!